### DISCUSSION TITLE
How to use sahi to infer the locally modified model

### DISCUSSION THREAD

**Discussion by SuperFDX** [obss/sahi#761] [Q&A] [answered]
Hello, thank you for the project code you provided. I am using the mmdetection library to modify my own model. Now I want to use sahi to infer my modified model. But I have seen the source code of sahi and need to provide the corresponding URL. However, for my own model trained locally, I cannot provide its URL, but can only provide the corresponding weight file path. What changes do I need to make on the sahi source code, Looking forward to your reply, thank you.

**Comment by devendraswamy** [comment]
Did you got the solution , If it is please let us know here , Thanking you in advance.

**Reply by fcakyon** [reply]
What is your issue?

**Comment by fcakyon** [comment, marked as answer]
You don't need to provide any URL. You must provide the weight path and config path for mmdetection models.

Just follow the mmdetection demo notebook: https://github.com/obss/sahi/blob/main/demo/inference_for_mmdetection.ipynb

**Reply by SuperFDX** [reply]
Thank you for your reply. I will study the notes you provided first.
